test/TestAppUtils/TestFrame.cs

```diff
@@ -169,20 +169,24 @@ private void OnViewScalingCheckBoxChanged(object sender, RoutedEventArgs e)
 
         private void SetRootGridSizeFromWindowSize()
         {
-#if USE_INTERNAL_SDK
+            var size = new Size(Window.Current.Bounds.Width, Window.Current.Bounds.Height);
+
+#if USE_INSIDER_SDK
             if (Windows.Foundation.Metadata.ApiInformation.IsTypePresent("Windows.UI.Xaml.XamlRoot"))
```

Comment:
Oh what I should have said is that IsTypePresent on insider builds can give you a false positive. In that it will tell you "yes, the type is present" but then you try to use it and it doesn't match the SDK you compiled against. I'll update my comment.